### PR TITLE
feat(rust): `project ticket` improvements

### PR DESCRIPTION
**Pull request by adrianbenavides**
- add support to `project ticket` and `project enroll` to produce/consume json encoded enrollment tickets
- show a warning when a ticket is generated with high values for both usage/expiration arguments. This won't prevent the user from generating the ticket.
- add plain output to `project ticket` 

image

### CODE REVIEW

File: implementations/rust/ockam/ockam_app_lib/src/lib.rs [hunk 1]
```diff
@@ -4,6 +4,7 @@
 //!
 //! It exposes C APIs that can be used by the frontend to interact with the application.
 //!
+#![recursion_limit = "256"]
```
Comment (davide-baldo) [inline implementations/rust/ockam/ockam_app_lib/src/lib.rs:7]: Why did you need to increase the limit?

**Review by davide-baldo — DISMISSED** [review]

File: implementations/rust/ockam/ockam_app_lib/src/lib.rs (hunk 1 above)
Comment (adrianbenavides) [reply implementations/rust/ockam/ockam_app_lib/src/lib.rs:7]: I keep getting a pretty ugly compilation error and that's what the compiler recommended to do to avoid the issue (basically increase that value)... It was caused by a third party library (aws-smithy I think). I was hoping that this would disappear after the recent deps updates we did, but it didn't.

**Review by davide-baldo — APPROVED** [review]
